server/application-server/src/main/java/de/tum/in/www1/hephaestus/chat/message/Message.java

```diff
@@ -0,0 +1,43 @@
+package de.tum.in.www1.hephaestus.chat.message;
+
+import jakarta.persistence.*;
+import java.time.OffsetDateTime;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.springframework.lang.NonNull;
+import de.tum.in.www1.hephaestus.chat.session.Session;
+
+@Entity
+@Table(name = "message")
+@Getter
+@Setter
+@ToString(callSuper = true)
+@NoArgsConstructor
+public class Message {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NonNull
+    @Column(name = "sent_at")
+    private OffsetDateTime sentAt = OffsetDateTime.now();
+
+    @NonNull
+    @Enumerated(EnumType.STRING)
+    private MessageSender sender;
+
+    @NonNull
+    @Column(name = "content")
```

Comment:
Same goes here, if you don't want to specify e.g. the String length or SQL type, you can omit `@Column`.